libs/vectorise/include/vectorise/uint/uint.hpp

```diff
@@ -21,6 +21,7 @@
 #include "vectorise/platform.hpp"
 
 #include <algorithm>
+#include <climits>
```

Comment:
What's this include for?

Reply:
Needed for it to built because it was cherry picked before the changes that were made